core/api/src/services/mongoose/schema.types.d.ts

```diff
@@ -90,6 +90,8 @@ interface AccountRecord {
   defaultWalletId: WalletId
   displayCurrency?: string
 
+  threadId?: string
```

Comment:
uhmm not sure about this.. do we really want to include this in account? what is the real purpose? (openai id?)

Reply:
it's the thread of communication you have (in case you decide to use the chatbot) with the open ai assistant

Reply:
I think the name could be more descriptive. My first interpretation was also an OS thread.

Perhaps its a `chatContextId` or `supportChatId` or something like that?